test/verify/check-sosreport

```diff
@@ -52,14 +50,15 @@ threads=2
             b.wait(lambda: b.eval_js('ph_find(".progress-bar").offsetWidth') > 0)
             b.wait_visible("#sos-download")
 
-        download_dir = tempfile.mkdtemp()
-        self.addCleanup(shutil.rmtree, download_dir)
-        b.cdp.invoke("Page.setDownloadBehavior", behavior="allow", downloadPath=download_dir)
+        for f in glob.glob(os.path.join(b.cdp.download_dir, '*')):
```

Comment:
OK, but more robust to do that with `os.listdir()`. But why does that need to be cleaned in the first place? Every test case starts with a fresh Browser and thus CDP object, and thus a new empty download dir?

Reply:
Right, this was thinko since it is created in CDP init and this it is clean for every test.